packages/broker/src/helpers/PayloadFormat.ts

```diff
@@ -0,0 +1,94 @@
+export interface PayloadFormat {
+    createMessage: (payload: string) => Message
+    createPayload: (content: any, metadata?: Metadata) => string
+}
+
+export interface Message {
+    content: any
```

Comment:
Not sure if this applies:
```suggestion
    content: Record<string, unknown>
```

Reply:
If I've understood correctly, the content can be an array. `Record<string, unknown>` would be too strict for that case. 

In `streamr-client` we  use object type, but it is not optimal either: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md.

Reply:
Fixed. Updated tests for cases where the content is an array or payload JSON is an array, or empty string. 

The publishHttp plugin now uses `PayloadFormat` to apply the validation logic.